CONFIGURE.md

```diff
@@ -287,6 +341,62 @@ Another option to define a state storage is to use the following properties in t
     nifi.state.storage.local.auto.persistence.interval=0 sec
 
 
+### Configuring Repositories
+
+Apache MiNiFi C++ uses three repositories similarily to Apache NiFi:
```

Comment:
typo:
```suggestion
Apache MiNiFi C++ uses three repositories similarly to Apache NiFi:
```

Reply:
Updated in 4e4434732dac8f91122fc910ea067c7c752283c6